tests/services-dnscrypt-proxy.nix

```diff
@@ -0,0 +1,29 @@
+{
+  config,
+  pkgs,
+  ...
+}:
+
+let
+  dnscrypt-proxy = pkgs.runCommand "dnscrypt-proxy-0.0.0" { } "mkdir $out";
+in
+{
+  services.dnscrypt-proxy.enable = true;
+  services.dnscrypt-proxy.package = dnscrypt-proxy;
+
+  ids.uids._dnscrypt-proxy = 123;
+  ids.gids._dnscrypt-proxy = 123;
```

Comment:
```suggestion
```

These should be set inside `modules/misc/ids.nix`

Reply:
Thank you, Today I learn. @Enzime